docs/aurora/data-science/jupyter.md

```diff
@@ -0,0 +1,136 @@
+# Running JupyterLab and Notebook on Aurora
+
+ALCF provides a [JupyterHub](https://jupyter.alcf.anl.gov/) for running JupyterLab and Notebook on Polaris and Sophia. While this service is not yet available on Aurora, you can still run JupyterLab and Notebook using ssh tunneling and port forwarding.
+
+---
```

Comment:
is there a reason to have several horizontal line dividers throughout this page? just curious

Reply:
not really. I am fine with removing any/all.